### PR TITLE
Final ginkgo migration

### PR DESCRIPTION
**Pull request by baude**
Completion of the migration from bats to ginkgo.  This includes:

* load
* mount
* pause
* port
* run_networking
* search

Note: build will be done within a different PR

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Please yank out pause_test.go, I've spruced it up and have added to it in #285

**Comment by mheon** [comment]
LGTM though I'd love to see the pause tests yanked out to make rebasing #285 easier

**Comment by mheon** [comment]
Merging once tests done

File: test/README.md [hunk 1]
```diff
@@ -1,106 +1,59 @@
...
 
-Integration tests are written in *bash* using the
-[bats](https://github.com/sstephenson/bats) framework.
-
-## Running integration tests
-
-### Containerized tests
-
-The easiest way to run integration tests is with Docker:
+### Installing ginko
```
Comment (TomSweeneyRedHat) [inline test/README.md]: nit for later ginko to gingko (sorry for not spotting sooner.)

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
required is of the opinion that CI is failing, but Fedora and CentOS and Travis are all passing.

**Pull request merged**